util/regexp-assemble/data/942180.data

```diff
@@ -25,11 +25,14 @@
 
 \d[\"'`]\s+[\"'`]\s+\d
 ^admin\s*?[\"'`]
-(?:/\*)+[\"'`]+\s?
-(?:/\*)+[\"'`]+\s?--
-(?:/\*)+[\"'`]+\s?#
-(?:/\*)+[\"'`]+\s?/\*
-(?:/\*)+[\"'`]+\s?{
+
+##! Escaped / 
```

Comment:
Why? regexp-assemble will take care of escaping if it's necessary.

Reply:
Then let's remove the comment?